installer/scripts/service_control

```diff
@@ -45,6 +46,14 @@ is_omsagent_running()
     fi
 }
 
+exit_if_agent_not_onboarded()
+{
+    if [ ! -f $OMSADMIN_CONF ]; then
+        echo "Warning: Agent is not onboarded. omsagent cannot be registered as a service."
+        exit 1
```

Comment:
I did build and test a shell bundle and it works find but you are right, emitting 0 would be better.